### ISSUE TITLE
Requesting "All Seasons" also requests the "Specials"

### ISSUE THREAD

**Issue by KaraokeStu** [tidusjar/Ombi#1745]
#### Ombi build Version:

V 3.0.2407

#### Update Branch:

Open Beta

#### Media Sever:

Plex

#### Operating System:

Ubuntu 16.04 (Linux 4.12.14-041214-generic #201709200843 SMP Wed Sep 20 12:46:23 UTC 2017)

#### Problem Description:

Adding a new series that has "Specials" (S00) available will request these too.
Many times these specials are unavailable, therefore the show never gets marked as available, even though the show is actually available.

This is being used with Sonarr.

#### Reproduction Steps:

Add the show "Better Call Saul" with "All Seasons".
Check Sonarr
Seasons 1, 2, 3 and "Specials" are added

**Comment by goldenpipes** [comment]
This seems correct to me.  

What i do, is if i don't want any specials i just go back into the request and remove the specials folder manually.

Maybe there could be check boxes next to each season before hitting approve, select the season(s) wanted then click approve?

**Comment by leram84** [comment]
if its one or the other, im betting way more people prefer not to have specials than ppl that want them (by default anyway)

**Comment by goldenpipes** [comment]
I just checked this again. For me when the whole show gets added, only the seasons are monitored in Sonarr it doesn't look to download the specials.  So in Ombi I still just manually remove the "specials" season and it marks the show as available.

**Comment by tidusjar** [comment]
@goldenpipes that is the expected result. @KaraokeStu can you confirm this is an issue?

**Comment by KaraokeStu** [comment]
It doesn't monitor the specials season, but it seems to mark them all as wanted. Since the majority of specials are hard to download or simply unavailable, it tends to be an issue.

**Comment by goldenpipes** [comment]
@tidusjar thats what i figured :)

Is it possible to just have the availability checker in ombi ignore the specials season altogether? Maybe like an "ignore specials" toggle on the request approval page? 

This way its still requested to your downloader, but not counted towards the availability of your show, unless you want it to.

**Comment by KaraokeStu** [comment]
@goldenpipes That seems to be a good solution

**Comment by tidusjar** [comment]
I'm pretty sure it ignores the specials, does it not?

**Comment by PearsonFlyer** [comment]
Requesting all for a show adds the specials episodes themselves as monitored, but the "season" as unmonitored.  It should be setting the episodes themselves as unmonitored as well as the season.

**Comment by anojht** [comment]
I can confirm the behavior described by @PearsonFlyer is what Ombi currently does. The specials season itself is unmonitored but all the special episodes are monitored when requesting all seasons inside Ombi. 

IMO, request all should request everything as the name suggests including specials or just all the seasons and specials could be handled through a request specials option. In Sonarr specials are treated as Season 0 not sure about other indexers, would implementing this option be tricky to do @tidusjar ?

**Comment by tidusjar** [comment]
I think it should not do the specials.

**Comment by anojht** [comment]
Yup that's cool so then the current behavior needs to be fixed and maybe we can look into adding a request specials option in the drop down sometime in the future?
